README.md

```diff
@@ -1,25 +1,37 @@
+# Ufoods app
+August 2017 Craft Academy Course Project
+
+We created an app which allows users to order a food in a various restaurants nearby. 
+You sign up by creating an account with an email and password. 
+After logging in you will be allowed to choose a restaurant and order a food with delivery.
+
 ##Getting Started
```

Comment:
To be sure it works well on Github, add a space between the title and the markdown for header 2. 
Example: ## Getting Started instead of ##Getting Started. 

It seems to work well in Rubymine, but it won't work when you comment on github, per example.